src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -0,0 +1,696 @@
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+
+            // Make the request
+            IPagedAsyncEnumerable<ListStringsResponse,string> response = await libraryServiceClient.ListStringsAsync();
+
+            // Iterate over all response items, lazily performing RPCs as required
+            foreach (string item in response)
+            {
+                // Do something with each item
+                Console.Writeline(item);
+            }
+
+            // Or iterate over fixed-sized pages, lazily performing RPCs as required
+            int pageSize = 10;
+            foreach (FixedSizePage<string> page in response.AsPages().WithFixedSize(pageSize))
+            {
+                // Do something with each page of items
+                Console.Writeline("A page of results:");
+                foreach (string item in page) {
+                    Console.Writeline(item);
+                }
+            }
+            // End snippet
+        }
+
+        public void ListStrings()
+        {
+            // Snippet: ListStrings(string,int?,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+
+            // Make the request
+            IPagedEnumerable<ListStringsResponse,string> response = libraryServiceClient.ListStrings();
+
+            // Iterate over all response items, lazily performing RPCs as required
+            foreach (string item in response)
+            {
+                // Do something with each item
+                Console.Writeline(item);
+            }
+
+            // Or iterate over fixed-sized pages, lazily performing RPCs as required
+            int pageSize = 10;
+            foreach (FixedSizePage<string> page in response.AsPages().WithFixedSize(pageSize))
+            {
+                // Do something with each page of items
+                Console.Writeline("A page of results:");
+                foreach (string item in page) {
+                    Console.Writeline(item);
+                }
+            }
+            // End snippet
+        }
+
+        public async Task AddCommentsAsync()
+        {
+            // Snippet: AddCommentsAsync(string,IEnumerable<Comment>,CallSettings)
+            // Additional: AddCommentsAsync(string,IEnumerable<Comment>,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatBookName("[SHELF]", "[BOOK]");
+            IEnumerable<Comment> comments = new []
+            {
+                new Comment
+                {
+                    Comment = ByteString.CopyFromUtf8(""),
+                    Stage = Stage.UNSET,
```

Comment:
Yes, this is a broken enum name.
This is now fixed, by porting the enum renaming code from https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/csharp/csharp_helpers.cc